CASMCMS-9423/CASMCMS-9448: Remove/update docs for removal of CPS/DVS, long-deprecated csm-config playbooks (#5994)
Cray-HPE/docs-csm: operations/iscsi_sbps/iscsi_sbps.md

* [Introduction](#introduction)
* [iSCSI SBPS solution details](#iscsi-sbps-solution-details)
* [Steps to achieve SBPS](#steps-to-achieve-sbps)
* [Glossary](#glossary)

## Introduction

iSCSI based boot content projection solution named Scalable Boot Content Projection Service (SBPS)

> is a boot content projection solution that replaces the former Cray

Reviewer: that replaces `projection via` the ~~former~~ Cray

Author: ```suggestion
is a boot content projection solution that replaces the Cray
```

[Content Projection Service (CPS)](../../glossary.md#content-projection-service-cps) and
[Data Virtualization Service (DVS)](../../glossary.md#data-virtualization-service-dvs).
SBPS projects boot content like `rootfs` and [Cray Programming Environment (CPE)](../../glossary.md#cray-programming-environment-cpe) images.
SBPS is aimed to offer better reliability, availability, security, ease and speed of deployment and ease of management than CPS/DVS.

> SBPS was introduced in CSM 1.6. CPS and DVS are removed in CSM 1.7.

Reviewer 2: ```suggestion
SBPS was introduced in CSM 1.6. In CSM 1.7, support is removed for projecting root filesystems and PE images using CPS and DVS.
```

The SBPS solution is spread across different components, including:

* [Boot Orchestration Service (BOS)](../../glossary.md#boot-orchestration-service-bos)

> * [Cray Operating System (COS)](../../glossary.md#cray-operating-system-cos)

Reviewer 2: FWIW I believe the remaining packages in COS/USS that helped support SBPS will be present in CSM 1.7.  We will them remove these components from USS in the following release in 2026.  Given that, it seems like this line could be removed.

Author: ```suggestion
```
